src/sidenav/SideNav.css

```diff
@@ -0,0 +1,33 @@
+.MuiPaper-root.side-nav-paper {
```

Comment:
I see there's a lot of custom CSS, is there something wrong with the default one?

Reply:
Nothing wrong, just had to override the default styles a bit to match our mocks.

http://shortn/_DJFATQ4frh vs https://material-ui.com/components/menus/#menulist-composition

Reply:
That's fair, I'm just worried we'll have to to a lot of customizations.
From a flow perspective, I think is better to use the default components, finish the MVP elements, and then teak them if needed.

But I'm open to do it this way too.

Reply:
That makes more sense, especially going forward. We can save these types of customizations till the end (though I think we may as well keep these ones, instead of redoing them later).